src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -298,6 +312,10 @@ namespace HVACVariableRefrigerantFlow {
             }
         }
 
+        // wait for air loop inputs
+        // figure out a way to check/initialize each TU so those that are zone equipment can be read in?
+        if (!DataAirLoop::AirLoopInputsFilled) return;
+
```

Comment:
Up for discussion as to how to read multiple TUs. All at once, as is done now, or one at a time which would probably avoid code like this. Chose to wait until air loop data has been read in (even if no air loops exist).. This allows all TUs to be read in and initialized all at once. This does need to be cleaned up eventually.